last-module-codecoverage/pom.xml

```diff
@@ -137,7 +152,7 @@
                             <parameters>
                                 <parameter>
                                     <name>lowerlimit</name>
-                                    <value>0</value>
+                                    <value>83</value>
```

Comment:
I suggest having 80% coverage level as minimum

Reply:
why not keep it as high as possible. if new code comes in, developer may decrease